doc/concept for the Ontohub 2.0 user interface

```diff
@@ -0,0 +1,60 @@
+und beschreibt dessen grundlegendes Konzept und Aufbau. Wir wollen somit 
+einen kurzen Überblick über die neue Benutzeroberfläche und der erreichbaren 
+Funktionen geben. Die Website “Ontohub.org” soll durch ein schlichtes und 
+übersichtliches Design verbessert werden. Der Nutzer soll durch dieses 
+neue Konzept alle wichtigen Funktionen schneller erreichen können. 
+Damit wird die Benutzerfreundlichkeit erhöht.
+
+Beim öffnen der Website soll der Nutzer auf eine übersichtliche Landingpage gelangen, 
+hier wird erstmals in angemeldete und unangemeldete User unterschieden.
+Für unangemeldete Nutzer wird auf der Landingpage die beliebtesten (nach Aufrufen, etc.) 
+und die kürzlich geupdatete Repositories/ Ontologien angezeigt. 
+Nach einer Anmeldung werden dazu auch noch die wichtigsten eigenen Repositories 
+(die der Nutzer beispielsweise als letztes aufgerufen hat) aufgelistet.
+Im oberen Bereich der Website soll eine Menüzeile angezeigt werden, 
+die zum Beispiel eine Suchfunktion, die je nach Ort (global, this organisation/user, repository) 
+suchen soll, und eine Login Funktion auf der rechten Seite bereitstellt. 
+Unter Anderem gibt es auch eine ausführliche Suchfunktion, die mit Hilfe einer ausklappbaren 
+Eingabezeile und Filterfunktion neben der einfachen Suche dargestellt werden soll.
+
+Es soll ein Standard View für alle Zugriffe vorhanden sein, die aus zwei modularen Tabs besteht.
+Auf der linken Seite wird der Filebrowser als Verzeichnisbaum dargestellt und auf der rechten Seite 
+die Beschreibung/ Inhalt von den Repositories/ OMS. 
+Dazu können die unten genannten Funktionen dazu geschaltet werden, sodass der User sich 
+seine eigene View erstellen kann und sowohl die Nutzung als auch die Navigation vereinfacht wird. 
+Zum Beispiel gibt es 3 Fenster, für diese kann der Nutzer dann beliebig entscheiden, 
+welche der verschiedene Funktionen er sehen möchte.
+
+Die Inhalte werden so dargestellt, 
+wie die Rechte des Owners es bestimmen. Als unangemeldeter Nutzer bekommt man beispielsweise 
+nur die grundlegende Übersicht und Zugriff auf die Daten und hat somit nur Leserechte.
+
+Es ist auch vorstellbar, dass im dargestellten Filebrowser 
+die Möglichkeit besteht, sich Favoriten anzeigen zu lassen.
+Funktionen wie die Suche, Commit, Compare oder Versions sollen mit Pop-Ups dargestellt werden, 
+es wird also das Bild ausgegraut und darüber wird der Funktionsdialog mit den Einstellungen dazu angezeigt. 
+Danach wird dies einfach wieder ausgeblendet. Diese Umsetzung soll die häufigen Seitenwechsel eingrenzen.
+
+Insgesamt soll ein einheitliches Design erstellt werden, in dem neue Funktionen schnell 
+hinzugefügt werden können, ohne dass die Seitenstruktur geändert wird.
+
+
+Funktionen als Tabs:
+- Filebrowser
+- Quelltext
+- Inhalt
+- Metadaten
+- Theorem Beweis
+- OMS Graph
+- Übersetzung
+
+Funktionen als PopUp:
+- Commit
+- erweiterte Suche mit Filter
+- Compare
+- Versionsmanagement/ Diff
+- schneller Zugriff auf Fehlermeldungen
+ 
```

Comment:
as own panel: history
in global menu bar: search with context dependent scope (all repos, this repo, this file, this OMS)

general panel structure
file browser - content - (optional) command
file browser: standard file browser, graph
content: README.md, OMS details, editor, diff, translation, translation-compare
commands: prove, translate, git with staged files (each staged file with show diff cation)/commit message+button/revert/add to staged 
mobile version: only one panel is shown, tabs for switching between panels

multi-file commit: mark staged files in file browser